openreview/venue/process/assignment_pre_process.py

```diff
@@ -3,14 +3,18 @@ def process_update(client, edge, invitation, existing_edge):
     domain = client.get_group(edge.domain)
     venue_id = domain.id
     submission_name = domain.content['submission_name']['value']
-    review_name = domain.content['review_name']['value']
```

Comment:
I would prefer to replace the name of the variable in the file instead of introducing a new variable `is_reviewer`